mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ProxyConfigurationResourceDefinition.java

```diff
@@ -124,15 +127,6 @@ public SimpleAttributeDefinitionBuilder apply(SimpleAttributeDefinitionBuilder b
         },
         AUTO_ENABLE_CONTEXTS("auto-enable-contexts", ModelType.BOOLEAN, new ModelNode(true)),
         BALANCER("balancer", ModelType.STRING, null),
-        CONNECTOR("connector", ModelType.STRING, null) {
```

Comment:
You can't just delete this attribute. We need to support translating legacy CLI scripts that reference this name.
This is easily handled via the ResourceDescriptor.addAlias(...) method.

Reply:
Fixed -- moved to DeprecatedAttribute.